collectd.conf

```diff
@@ -1,5 +1,7 @@
 Interval 60
 
+Hostname ""
```

Comment:
Adding the comment Igor mentioned about why this is left blank can be helpful to anyone contributing in the future, not necessarily our users.

Reply:
Done.